src/NumericalAlgorithms/Interpolation/Callbacks/ObserveSurfaceIntegrals.hpp

```diff
@@ -85,7 +85,12 @@ struct ObserveSurfaceIntegrals {
     });
 
     // Write to a file.
-    // Do we need file locking?
+    // Currently there is no file locking.
+    //
+    // Once this PR (and another one of Nils Deppe that solves the issue
+    // https://github.com/sxs-collaboration/spectre/issues/1198) has been
+    // merged, another PR will replace the file-writing here with a call to
+    // ContributeReductionData, and this will solve the file locking problem.
```

Comment:
"This PR" doesn't mean much in the codebase, also I think this is not the place to refer to specific people (that's what assigning issues is for). I suggest something along the following lines:

> Currently there is no file locking. When issue https://github.com/sxs-collaboration/spectre/issues/1198 is resolved we will replace the file-writing here with calls to the observer infrastructure, which takes care of the file locking.